src/provider/view/components/NotificationGroup/NotificationGroup.tsx

```diff
@@ -1,42 +1,56 @@
 import * as React from 'react';
+import {CSSTransition, TransitionGroup} from 'react-transition-group';
 
 import {NotificationCard} from '../NotificationCard/NotificationCard';
 import {StoredNotification} from '../../../model/StoredNotification';
-import {CloseButton} from '../CloseButton/CloseButton';
-import {Actionable} from '../../containers/NotificationCenterApp';
+import {Actionable} from '../../../store/Actions';
 import {RemoveNotifications} from '../../../store/Actions';
+import {CircleButton} from '../CircleButton/CircleButton';
 
-export interface NotificationGroupProps extends Actionable {
+import './NotificationGroup.scss';
+
+interface Props extends Actionable {
     // Group name
     name: string;
     // Notifications in this group
     notifications: StoredNotification[];
 }
 
-export function NotificationGroup(props: NotificationGroupProps) {
-    const {notifications, storeDispatch} = props;
+export function NotificationGroup(props: Props) {
+    const {notifications, storeDispatch, name} = props;
     const handleClearAll = () => {
         storeDispatch(new RemoveNotifications(notifications));
     };
+
     return (
         <div className="group">
-            <div className="header">
-                <div className="title">
-                    {props.name.toUpperCase()}
-                </div>
-                <CloseButton onClick={handleClearAll} />
+            <div className="title">
+                <span className="app">
+                    <span className="name single-line">{name}</span>
+                    <span className="count">{notifications.length}</span>
+                </span>
+                <CircleButton type="close" size="small" onClick={handleClearAll} alt="Clear notifications" />
             </div>
-            <ul>
+            <TransitionGroup className="notifications" component="ul">
                 {
-                    notifications.map((notification, i) => {
+                    notifications.map((notification) => {
                         return (
-                            <li key={i + notification.id}>
-                                <NotificationCard notification={notification} storeDispatch={storeDispatch} />
-                            </li>
+                            <CSSTransition
+                                key={notification.id}
+                                timeout={{
+                                    enter: 300,
+                                    exit: 300
```

Comment:
Can we make these a bit quicker? Any animations should be really non-intrusive.

If making the animation makes things look weird, I'd prefer to remove, or go with a fade-type animation.

Reply:
Changed to 200ms